src/cirrocumulus/scripts/cirro_scanner.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# cirro_scanner.sh scans the mounted buckets to find cirro data. Creates a cirro dataset when *.zarr folder
+# is found and is not yet in the existing cirro datasets.
+
+if [[ $# -ne 1 ]]; then
+  echo "Usage: $0 <true/false>"
+  exit 1
+fi
+
+readonly LOG_IN="${1}"
+if [[ "${LOG_IN}" == "false" ]]; then
+  echo "user is not logged in"
+  exit 0
+fi
+
+wb resource mount
+
+# Define the function to check if folder path exists in the JSON array
+dataset_exists() {
+    local url="$1"
+    local array_json=$(curl -s localhost:3000/api/datasets)
+    local exists=$(echo "$array_json" | jq -e --arg path "$url" '.[] | select(.url == $path)' | wc -l)
```

Comment:
Why set the variable just to echo it? How about replacing these two lines with:

```
echo "$array_json" | \
  jq -e --arg path "$url" '.[] | select(.url == $path)' | \
  wc -l
```

The function should be renamed and have a comment indicating that it outputs a numerical value. "dataset_exists" implies a boolean return.